packages/config/types/build.d.ts

```diff
@@ -58,6 +58,8 @@ export interface NuxtConfigurationBuild {
   templates?: any
   terser?: TerserPluginOptions | boolean
   transpile?: (string | RegExp)[]
-  useForkTsChecker?: { [key: string]: string } | boolean // TBD - Couldn't find typedefs for the forkTsCheckerWebpackPlugin options
+  typescript?: {
+    typeCheck?: { [key: string]: any} | boolean // TBD - Couldn't find typedefs for the forkTsCheckerWebpackPlugin options
```

Comment:
```suggestion
    typeCheck?: { [key: string]: any } | boolean // TBD - Couldn't find typedefs for the forkTsCheckerWebpackPlugin options
```

Reply:
Oops @danielroe , could you accept this last suggestion so I can aproove the PR ? Thanks.